static/include/components/apis/overrides/protos/button.Push.md

```diff
@@ -0,0 +1,2 @@
+Push the button.
+Blocks until done or until a new operation cancels this one.
```

Comment:
```suggestion
```

Reply:
Ooh okay it says this in the code comment--should it not?

Reply:
Np!